pkg/common/connectionmanager/search.go

```diff
@@ -63,6 +63,8 @@ func (cm *ConnectionManager) WhichVCandDCByNodeID(ctx context.Context, nodeID st
 	if searchBy == FindVMByUUID {
 		klog.V(3).Info("WhichVCandDCByNodeID by UUID")
 		myNodeID = strings.ToLower(nodeID)
+	} else if searchBy == FindVMByIP {
```

Comment:
this is a nit more than anything, but I really dislike `else if` statements in go 😄, can you make this a `switch`?

Reply:
Yup